ha/setup/ha_setup.py

```diff
@@ -356,8 +356,9 @@ def process(self):
                     self._confstore.set(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
                 except Exception as e:
                     Log.error(f"Cluster creation failed; destroying the cluster. Error: {e}")
-                    output = self._execute.run_cmd(const.PCS_CLUSTER_DESTROY)
-                    Log.error(f"Cluster destroyed. Output: {output}")
+                    # output = self._execute.run_cmd(const.PCS_CLUSTER_DESTROY)
+                    # Log.error(f"Cluster destroyed. Output: {output}")
+                    self._cluster_manager.cluster_controller.destroy_cluster()
                     # Delete the node from nodelist if it was added in the store
                     if self._confstore.key_exists(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}"):
```

Comment:
`self._cluster_manager.cluster_controller.destroy_cluster()`
Also need to use it in cleanup

Reply:
Yeah. Got this now. Modified